src/errors/IncorrectFileTypeError.ts

```diff
@@ -0,0 +1,6 @@
+export default class IncorrectFileTypeError extends Error {
+  constructor() {
+    super();
```

Comment:
Some personal preference and opinionated view... so feel to ignore :)

This is exactly why I think OOP with Ts (Js) or in general is not so great because we end up creating 5 classes to use polymorphism and override a message. This could all be avoided in the first place if we had written a simple function passing a string.
It creates more code to maintain, etc..